src/FlexDayFacade.php

```diff
@@ -5,12 +5,12 @@
 use Illuminate\Support\Facades\Facade;
 
 /**
- * @see \Tipoff\FlexScheduling\FlexScheduling
+ * @see \Tipoff\FlexScheduling\FlexDayFacade
  */
-class FlexSchedulingFacade extends Facade
+class FlexDayFacade extends Facade
 {
     protected static function getFacadeAccessor()
     {
-        return 'flex-scheduling';
+        return 'flexdays';
```

Comment:
I want to confirm if the return value is ok? @drewroberts

Reply:
Yes, that works for me.